spec/menu_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu'
+
+describe Menu do
+
+  it "can add a dish to menu array" do
+    subject.add_dish("fish and chips", 5)
+    expect(subject.menu.length).to eq 6
```

Comment:
This is hard to understand due to the magic number. One way you could improve this is to allow your Menu class to be passed a hash of menu items when its initialised. You can then create a test hash of items that you can refer to during testing.